### DISCUSSION TITLE
Add album art to books in BookPlayer

### DISCUSSION THREAD

**Discussion by loofi** [TortugaPower/BookPlayer#564] [Ideas]
I would first like to thank you for an excellent and a no-fuzz app.

It would be really helpful if the app had an option to add album art. Not all books imported comes with this and be able to process everything on iOS would make it perfect.

Thank you.

**Comment by pichfl** [comment]
A year ago I would have (and also have) answered that Book*Player* is for playing books not *editing* them. However, we are planning to provide a way to group single files virtually into a single book and with that we'll also provide a way to edit meta data, which includes artworks as well.

We still recommend binding books on your computer [as described in our Wiki](https://github.com/TortugaPower/BookPlayer/wiki/FAQ#how-to-combine-mp3-files-into-m4b-audio-books). Trying to do the same on a phone would most likely melt it as it. Single file books are much easier to handle in the first place and there are free tools to make them.

**Reply by loofi** [reply]
Thank you for the swift reply. 
Its great to hear that this will be implemented. Looking forward it!

Great work:)

**Reply by frehley666** [reply]
Thank you for fixing this...I thought I was going crazy when my books stopped showing the covers all of a sudden...keep up the great work!!!

**Reply by pichfl** [reply]
@frehley666 I think you got a bit lost. For reference, we had bug #556 which was fixed in #560 and will be / was released in BookPlayer 3.12.

This discussion is talking about editing books in BookPlayer which we will do some day in the future 🙂
